### PR TITLE
Fix bug of _prepare_4d_attention_mask

### PR DESCRIPTION
**Pull request by jiqing-feng**
Hi @ArthurZucker @younesbelkada . Since `_prepare_4d_attention_mask` is no longer a member function of `AttentionMaskConverter`, I directly import `_prepare_4d_attention_mask` from `modeling_attn_mask_utils`. Would you please help to review it? Thx!

BTW, the failed CIs are not related to my changes

### CODE REVIEW

File: src/transformers/models/llama/modeling_llama.py [hunk 1]
```diff
@@ -80,7 +84,7 @@ def _expand_mask(mask: torch.Tensor, dtype: torch.dtype, tgt_len: Optional[int]
     warnings.warn(
         "Calling `transformers.models.llama.modeling_llama._prepare_4d_attention_mask` is deprecated and will be removed in v4.37. Use `transformers.modeling_attn_mask_utils.AttentionMaskConverter._prepare_4d_attention_mask"
     )
-    return AttentionMaskConverter._prepare_4d_attention_mask(mask=mask, dtype=dtype, tgt_len=tgt_len)
+    return _prepare_4d_attention_mask(mask=mask, dtype=dtype, tgt_len=tgt_len)
```
Comment (ArthurZucker) [inline src/transformers/models/llama/modeling_llama.py:87]: this is deprecated anyway, but good catch.

**Review by ArthurZucker — APPROVED** [review]
The failing CI is related to the styling, make style should help

**Review by younesbelkada — COMMENTED** [review]
Nice catch! I left one comment, for the failing CI can you try to merge with main and make sure you have `ruff==0.1.5` installed when running `make fixup` ?

**Review by younesbelkada — APPROVED** [review]
Thanks!

**Pull request merged**